### ISSUE TITLE
Fix Meteor package imports when useRelativePaths is true

### ISSUE THREAD

**Issue by rhettlivingston** [Galooshi/import-js#293]
Meteor uses the convention of prepending "meteor/" to Meteor package names to signal to the build environment that the reference is to a Meteor package as opposed to an NPM package. This conflicts with the existing strategy invoked by the useRelativePaths=true option (the default). Instead of the desired

```js
import { Meteor } from 'meteor/meteor';
```

importjs is outputting

```js
import { Meteor } from './meteor/meteor';
```

An exception similar to the one already present for node modules probably needs to be added at https://github.com/Galooshi/import-js/blob/a8a3ce020cd650a4993d2bff14ce6e953eea7c3b/lib/Configuration.js#L271 to achieve the desired result.

This issue was created as a result of discussion in #289.

**Comment by rhettlivingston** [comment]
Simply adding the line referring to 'meteor/' below at lib/Configuration.js#L271 seems to fix the problem.
```js
    if (this.get('useRelativePaths', { fromFile: relativeFilePath }) &&
      !relativeFilePath.startsWith('meteor/') &&
      !relativeFilePath.startsWith('node_modules')) {
```
All tests passed with this addition and it fixed a test in my work for #289 that was failing due to the problem. So, I have some confidence in its effectiveness.

Are there tests that are explicitly targeted towards verifying the effectiveness of the above-noted line containing 'node_modules' that I could duplicate and adjust to more explicitly verify the effectiveness of the 'meteor/' line? If not, are we good without more checks or do I need to put one somewhere?

**Comment by trotzig** [comment]
I think that's good enough. If you look at a8a3ce02 (the commit that introduces the `node_modules` exception), there's [a test that you can use as a template](https://github.com/Galooshi/import-js/commit/a8a3ce02#diff-da30310350dcd91444d26504857a03c0R1252). It's in `Importer_test.js` which is a bit of a pain to work with due to its size. We have a long-standing plan to break it up, but we haven't prioritized it yet.

**Comment by rhettlivingston** [comment]
This issue was resolved by the changes in 04cc6c2 and e343494 which were merged to master via 629e49b.

As noted before, I need to work on my commit messages. Had to go read the github docs too. :frowning:
